jest.config.js

```diff
@@ -14,5 +16,9 @@ module.exports = {
     transform: {
         '^.+\\.tsx?$': 'ts-jest'
     },
-    testMatch: ['**/*.*Test.ts']
+    testMatch: ['**/*.unitTest.ts'],
```

Comment:
Mostly curious why the change to just `**/*.unitTest.ts` ?

Reply:
So it only includes the unit tests in the test explorer.
Before:
![image](https://user-images.githubusercontent.com/13391709/211863950-9090c58d-dde5-40dc-b8e4-45cdcd91af31.png)
After:
![image](https://user-images.githubusercontent.com/13391709/211859162-3ce54d44-bd6e-432b-8540-39d6c401c997.png)